mlrun/common/schemas/model_monitoring/model_endpoints.py

```diff
@@ -131,6 +131,15 @@ def _uid_to_str(cls, v):  # noqa: N805
             return str(v)
         return v
 
+    @validator("mode", pre=True, always=True)
+    def _set_mode_based_on_endpoint_type(cls, v, values):  # noqa: N805
```

Comment:
I suggest
https://github.com/mlrun/mlrun/blob/cc74f73a862905d1eb2adcbb80c8c7fc72550f00/mlrun/model_monitoring/applications/results.py#L82
Instead of `noqa: N805`

Reply:
The example is irrelevant, but the comment is still relevant
https://docs.pydantic.dev/1.10/usage/validators/
> validators are "class methods", so the first argument value they receive is the UserModel class, not an instance of UserModel.